README.md

```diff
@@ -45,13 +45,36 @@ Note that IntelliJ-specific paths can be found with [this guide to IntelliJ stor
 
 ### Dependencies
 
-Dependencies are managed by `dep`. Create a `pkgset` with `gvm pkgset create amanar`, then switch to it with `gvm pkgset use amanar`. The pre-GVM `GOPATH` should be added to the `GOPATH` of `gvm pkgenv amanar`. The `/bin` directory may be `PATH`. `dep ensure` should be run to install dependencies. Finally, run `go install` to build the binary.
+```bash
```

Comment:
The eventual goal is to move to Go modules with the release of Go 1.13 sometime this month. The `README` should be left as it is until then.